### PR TITLE
Pop over

### PR DESCRIPTION
**Pull request by JohnTheo13**
Popovers ready for review

### CODE REVIEW

**Comment by JohnTheo13** [comment]
@xDae Can you check the Popovers and let me know the changes you want me to make

**Comment by xDae** [comment]
thanks por the PR! @JohnTheo13 🎉
I will review it and merge :)

File: src/Popovers/default-theme.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// @$flow
```
Comment (xDae) [inline src/Popovers/default-theme.js]: ``// @flow``

File: src/Popovers/Popover.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+const Popover = ({ title, children, ...props }) => (
+  <StyledPopover {...props}>
+    <div />
+    {title && <Heading as="h3">{title}</Heading>}
+    <div>{children}</div>
+  </StyledPopover>
+);
+
+Popover.defaultProps = {
+  title: undefined
```
Comment (xDae) [inline src/Popovers/Popover.js]: I think is better to have an empty string as default ?

File: src/Popovers/default-theme.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+import { sassRgba } from '../utils/src/sassRgba';
+import { black, white } from '../utils/default-theme';
+
+export const zindexPopover = '1060';
```
Comment (xDae) [inline src/Popovers/default-theme.js]: ``zIndexPopover``

**Comment by JohnTheo13** [comment]
@xDae changes applied

**Comment by xDae** [comment]
can you put the package-lock.json in the .gitignore to avoid the file conflicts ?

**Pull request merged**